src/Node_verify.cpp

```diff
@@ -317,10 +317,25 @@ void Node::verify_vdf(std::shared_ptr<const ProofOfTime> proof, const uint32_t c
 			max_iters = std::max(max_iters, segments[i].num_iters);
 		}
 		if(have_sha_ni) {
-			for(uint32_t j = 0; j < num_lanes; ++j)
+			for(uint32_t j = 0; j < num_lanes; j += 2)
 			{
 				const uint32_t i = chunk * batch_size + j;
-				recursive_sha256_ni(point[j].data(), segments[i].num_iters);
+				uint8_t hashx2[32 * 2];
+
+				if(j + 1 >= num_lanes) {
+					recursive_sha256_ni(point[j].data(), segments[i].num_iters);
```

Comment:
In this case you can just compute a dummy hash for the overflow segment, like I do with AVX2.

Reply:
Should be achieved in Rev02 commit.